MvvmCross/Base/MvxMainThreadAsyncDispatcher.cs

```diff
@@ -7,8 +7,17 @@
 
 namespace MvvmCross.Base
 {
-    public abstract class MvxMainThreadAsyncDispatcher : MvxMainThreadDispatcher, IMvxMainThreadAsyncDispatcher
+    public class MvxMainThreadAsyncDispatcher : MvxMainThreadDispatcher, IMvxMainThreadAsyncDispatcher
     {
+        private int managedThreadId;
+
+        public MvxMainThreadAsyncDispatcher(int managedThreadId)
```

Comment:
Where does this ctor get set?

Reply:
Doesn't seem like this is set anywhere as @martijn00 points out.